include/learning_mode.h

```diff
@@ -0,0 +1,20 @@
+/**
+* @brief decision_modes definition.
+*
+* @file decision_modes.h
+* @author Chenxi Zhao
+* @date 2020-02-13
+*/
+#pragma once
+
+namespace reinforcement_learning {
+
+  enum learning_mode {
+    ONLINE_MODE = 0,

```

Comment:
Change to ONLINE and IMITATION

Reply:
Please check constants.h - config values constants are defined there.

Here I would prefer using ONLINE_MODE as enum reference so the code would be easier to read.

E.g live_model_impl.cc line 93

`    if (_learning_mode == IMITATION_MODE) {
      // Imitaion mode is not supported here at this moment
      return error_code::not_supported;
    }
`